_components/sidenav.md

```diff
@@ -7,8 +7,8 @@ lead: Intro text on what is included in this section and how to use it. No more
 
 <div class="preview">
   
-  <h2 class="usa-heading">Sidenav</h2>
-
+  <h3 class="usa-heading">1. Single level</h3>
```

Comment:
@mollieru we're not numbering anywhere else on the site where we have multiple versions of a component. Why are we starting this convention? Shouldn't it be consistent with the rest?